### ISSUE TITLE
npx degit failed due to missing #!/usr/bin/env node line

### ISSUE THREAD

**Issue by xywang68** [Rich-Harris/degit#274]
```
npx: installed 1 in 0.681s
/home/xywang68/.npm/_npx/805/bin/degit: 1: use strict: not found
/home/xywang68/.npm/_npx/805/bin/degit: 3: Syntax error: "(" unexpected
```
added
```
#!/usr/bin/env node line
```
to  bin/degit solve the problem.

**Comment by abannsunny** [comment]
faced the same issue solved it exactly as  @xywang68  did.

Giving a bit more context:

Stumbled upon similar issue:
https://github.com/domenic/worm-scraper/issues/5#issuecomment-321033496

Stack overflow answer
https://stackoverflow.com/a/34354713

**Comment by alsotang** [comment]
the same problem happened to me

**Comment by nashaad** [comment]
Install degit `npm install -g degit`

Edit degit, add this line `#!/usr/bin/env node` at the top of the file
```
nano /usr/local/bin/degit
```

**Comment by keshprad** [comment]
I'm also getting the same issue. I did the same thing @nashaad did and it works for me.

**Comment by gwelr** [comment]
Had the same issue today (was working fine two days ago). 
I noticed I was not using the latest npx version, so I upgraded npm 7.7.6. 

As proposed above, I edited degit script and added the following line at the top of the file: 

```sh
#!/usr/bin/env node
```
It fixed the issue. I'm on ubuntu 20.04.

**Comment by AlexMercedCoder** [comment]
Yeah I was teaching a class which I guess was 30 minutes after the update and had the same issue with all my students. Turned it into an impromptu lesson on reading github issues (then they just cloned the starter code). Sounds like a quick fix so hopefully the Saturday class will go smoother :).

**Comment by kazuma1989** [comment]
@Rich-Harris Thank you for quick fixes!

- https://github.com/Rich-Harris/degit/commit/07222570613a5326db11269c2f8dc0324add765f#diff-f9469b2caca44d79b8f2f7c81296a7b2bf9b19e25fb3e1450b9f4f3bf4117e2b
- https://github.com/Rich-Harris/degit/commit/64b80577acf3313b669840f7452800ee8d09fbf3#diff-f9469b2caca44d79b8f2f7c81296a7b2bf9b19e25fb3e1450b9f4f3bf4117e2b

Now v2.8.4 works fine 🎉

**Comment by AlexMercedCoder** [comment]
YES, Thank you!

**Comment by YogliB** [comment]
Hi! Thanks for flagging this. I just checked the latest release (3.6.6) and master, and `npx degit` runs fine — the `degit` bin wrapper includes `#!/usr/bin/env node` now. Looks like this was resolved at some point, so I'm closing it.
